src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -47,14 +47,22 @@ class ValidateActor(wdlSource: WdlSource, wdlJson: WdlJson)
 
   private def validateWorkflow(sentBy: ActorRef): Unit = {
     logger.info(s"$tag for $sentBy")
-    val futureValidation: Future[Unit] = for {
+    val futureValidation: Future[NamespaceWithWorkflow] = for {
       namespaceWithWorkflow <- Future(NamespaceWithWorkflow.load(wdlSource))
-      inputs <- Future(wdlJson.parseJson).map(_.asJsObject.fields)
-      coercedInputs <- Future.fromTry(namespaceWithWorkflow.coerceRawInputs(inputs))
```

Comment:
It might have been easier to have a method that returns a `Future` for the validated inputs if there are inputs, or `Future.successful(())` if not.